ADONET/AdoNetFundamentals/DLL/Repositories/ProductRepository.cs

```diff
@@ -0,0 +1,134 @@
+﻿using AdoNetFundamentals.Entities;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdoNetFundamentals.DLL.Repositories
+{
+    public class ProductRepository : IProductRepository
+    {
+        public SqlCommand GetCommand(string query)
+        {
+            var connection = new SqlConnection(AdoNetDbContext.ConnectionString);
+            var command = new SqlCommand(query, connection);
+
+            connection.Open();
+
+            return command;
+        }
+
+        public Product MapProduct(SqlDataReader reader)
```

Comment:
Such mappers that accept `SqlDataReader` or some other ADO.Net API should marked as private to prevent public exploits.